pymess/locale/cs/LC_MESSAGES/django.po

```diff
@@ -17,6 +17,14 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 2.2.1\n"
 
+#: backend/__init__.py:50
+msgid "the number of upload attempts exceeded the limit"
+msgstr ""
+
+#: backend/__init__.py:53
+msgid "the age of the message exceeds the send limit"
+msgstr ""
+
```

Comment:
I've added a translation by `cd pymess`, `django-admin makemessages` -a and `django-admin compilemessages`